src/awesome_map/EventSynthesizer.js

```diff
@@ -171,11 +171,11 @@ define(function(require) {
         this._transforming = false;
 
         /**
-         * Depending on environment, trigger resize on 'resize' or 'orientationChange'
-         * @type {Boolean}
+         * Trigger resize on 'resize'.
+         * @type {string}
          * @private
          */
-        this._resizeEventType = (typeof dependencies.getWindow().cordova !== 'undefined') ? 'orientationchange' : 'resize';
+        this._resizeEventType = 'resize';
```

Comment:
Not sure if we event need this. Could just inline the string as it's now constant.